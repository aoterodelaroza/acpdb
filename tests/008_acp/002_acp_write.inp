## check: 002_acp_write.out -a1e-10
## check: 002_acp_write.acp -a1e-10
## labels: regression quick

acp load bleh ../zz_source/minis.bsip
acp write bleh
acp write bleh 002_acp_write.acp
