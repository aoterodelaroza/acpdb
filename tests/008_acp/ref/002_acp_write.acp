-H 0
H 1 0
l
7
2 8.000000000000000e-02 -9.602266107482001e-03
2 1.000000000000000e-01 5.512770623187100e-02
2 1.200000000000000e-01 -7.539707136845800e-02
2 2.000000000000000e-01 8.376071405251299e-02
2 3.000000000000000e-01 -7.999789084717600e-02
2 7.000000000000000e-01 3.199679965913700e-02
2 4.000000000000000e+00 1.182111145935308e+00
s
5
2 8.000000000000000e-02 -1.835382073440000e-03
2 1.800000000000000e-01 -2.262448742196900e-02
2 5.000000000000000e-01 5.080916552346200e-02
2 6.000000000000000e-01 5.320275244223600e-01
2 1.800000000000000e+00 -1.862002126602946e+00-B 0
B 2 0
l
6
2 8.000000000000000e-02 3.079608278258000e-03
2 1.000000000000000e-01 -9.479500244160000e-04
2 2.200000000000000e-01 -6.628591408803000e-03
2 2.400000000000000e-01 -3.168388193164900e-02
2 5.000000000000000e-01 3.794506303070200e-02
2 1.200000000000000e+00 -2.522345343321740e-01
s
3
2 1.200000000000000e-01 2.396909849567380e-01
2 5.000000000000000e-01 -3.856815336643100e-01
2 4.000000000000000e+00 3.212214973747380e-01
p
2
2 8.000000000000000e-02 -7.815710493250699e-02
2 1.400000000000000e-01 1.517099090459450e-01-C 0
C 2 0
l
8
2 8.000000000000000e-02 -2.618096088330000e-03
2 1.000000000000000e-01 2.912160366525300e-02
2 1.200000000000000e-01 -5.193077357814600e-02
2 2.000000000000000e-01 8.199428242905400e-02
2 3.000000000000000e-01 -5.899034116794000e-02
2 4.000000000000000e-01 -1.717704783840850e-01
2 7.000000000000000e-01 2.890134876659910e-01
2 1.600000000000000e+00 -6.690791093990300e-01
s
3
2 1.000000000000000e-01 2.641795942408900e-02
2 3.000000000000000e-01 -8.011918062043601e-02
2 4.000000000000000e+00 5.135931758916020e-01
p
3
2 8.000000000000000e-02 -4.102337580953200e-02
2 1.400000000000000e-01 1.543425571225740e-01
2 1.600000000000000e-01 5.426003157596000e-03-N 0
N 2 0
l
6
2 8.000000000000000e-02 3.916089048575000e-03
2 1.200000000000000e-01 -1.209604524141300e-02
2 2.000000000000000e-01 7.052628297295000e-03
2 2.200000000000000e-01 8.680367925295000e-03
2 7.000000000000000e-01 -3.205701578810170e-01
2 3.000000000000000e+00 2.883822235932840e-01
s
2
2 1.000000000000000e-01 7.318190532596799e-02
2 5.000000000000000e-01 -4.142680556226500e-02
p
3
2 8.000000000000000e-02 -8.742704786934900e-02
2 1.400000000000000e-01 2.438428885551580e-01
2 5.000000000000000e-01 -9.068615631759799e-02-O 0
O 2 0
l
6
2 8.000000000000000e-02 -2.521325315554100e-02
2 1.000000000000000e-01 1.154773505943930e-01
2 1.200000000000000e-01 -1.313492750327370e-01
2 2.000000000000000e-01 7.039514688507301e-02
2 4.000000000000000e-01 -6.218435634278500e-02
2 1.000000000000000e+00 -4.934296935495230e-01
s
3
2 8.000000000000000e-02 5.183947158215900e-02
2 4.000000000000000e-01 -7.705302140412100e-02
2 4.000000000000000e+00 7.657587065816000e-03
p
5
2 8.000000000000000e-02 -4.410843289720700e-02
2 1.400000000000000e-01 1.045546080289900e-02
2 1.600000000000000e-01 1.378003771808770e-01
2 5.000000000000000e-01 -9.853965850886000e-02
2 4.000000000000000e+00 5.474151274499010e-01-F 0
F 2 0
l
7
2 8.000000000000000e-02 -7.651018085518000e-03
2 1.000000000000000e-01 2.222382002671200e-02
2 1.600000000000000e-01 -4.503808766972400e-02
2 2.800000000000000e-01 5.445365585453900e-02
2 3.000000000000000e-01 5.587047966493300e-02
2 7.000000000000000e-01 -2.692779500683450e-01
2 8.000000000000000e-01 -2.184186297321210e-01
s
2
2 8.000000000000000e-02 -3.049268533221500e-02
2 4.000000000000000e+00 8.257751939994901e-02
p
1
2 1.800000000000000e-01 5.537506810589800e-02-Si 0
Si 2 0
l
3
2 8.000000000000000e-02 5.411854811307000e-03
2 1.800000000000000e-01 -5.225981041500000e-05
2 2.000000000000000e-01 -3.908705111282600e-02
s
1
2 8.000000000000000e-02 -1.539039298301430e-01
p
2
2 1.800000000000000e-01 6.035911192153400e-02
2 3.000000000000000e+00 4.430728734537440e-01-P 0
P 2 0
l
4
2 8.000000000000000e-02 2.079212916411000e-03
2 1.400000000000000e-01 -2.222868567617000e-03
2 2.600000000000000e-01 -6.195693944512100e-02
2 2.800000000000000e-01 -3.169678225752600e-02
s
1
2 8.000000000000000e-02 1.021454873630330e-01
p
2
2 8.000000000000000e-02 -5.429454457614700e-02
2 2.000000000000000e-01 1.155644566911130e-01-S 0
S 2 0
l
5
2 8.000000000000000e-02 4.085452180255000e-03
2 1.200000000000000e-01 -1.957113499780000e-03
2 2.200000000000000e-01 4.095989244621000e-03
2 4.000000000000000e-01 -2.787783136850390e-01
2 5.000000000000000e-01 -4.968880762359400e-02
s
2
2 8.000000000000000e-02 2.903200571150390e-01
2 4.000000000000000e+00 -2.989405707391810e-01
p
3
2 8.000000000000000e-02 -4.775630073446100e-02
2 1.000000000000000e-01 -2.822775036148480e-01
2 2.600000000000000e-01 6.618928120329290e-01-Cl 0
Cl 2 0
l
4
2 8.000000000000000e-02 3.474205328673000e-03
2 1.000000000000000e-01 -9.928237398647000e-03
2 1.600000000000000e-01 3.173939248308600e-02
2 3.000000000000000e-01 -2.030771774724720e-01
s
1
2 1.200000000000000e-01 4.205976024310200e-02
p
2
2 8.000000000000000e-02 -4.737429474226200e-02
2 1.200000000000000e-01 1.074338339555610e-01