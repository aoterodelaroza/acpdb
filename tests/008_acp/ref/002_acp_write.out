%% verbose
%% acp load bleh ../zz_source/minis.bsip
* ACP LOAD bleh

%% acp write bleh
* Terms for ACP bleh
| id | atom | l | exponent | coefficient |
| 0 | H | l | 0.08 | -0.009602266107 |
| 1 | H | l | 0.1 | 0.05512770623 |
| 2 | H | l | 0.12 | -0.07539707137 |
| 3 | H | l | 0.2 | 0.08376071405 |
| 4 | H | l | 0.3 | -0.07999789085 |
| 5 | H | l | 0.7 | 0.03199679966 |
| 6 | H | l | 4 | 1.182111146 |
| 7 | H | s | 0.08 | -0.001835382073 |
| 8 | H | s | 0.18 | -0.02262448742 |
| 9 | H | s | 0.5 | 0.05080916552 |
| 10 | H | s | 0.6 | 0.5320275244 |
| 11 | H | s | 1.8 | -1.862002127 |
| 12 | B | l | 0.08 | 0.003079608278 |
| 13 | B | l | 0.1 | -0.0009479500244 |
| 14 | B | l | 0.22 | -0.006628591409 |
| 15 | B | l | 0.24 | -0.03168388193 |
| 16 | B | l | 0.5 | 0.03794506303 |
| 17 | B | l | 1.2 | -0.2522345343 |
| 18 | B | s | 0.12 | 0.239690985 |
| 19 | B | s | 0.5 | -0.3856815337 |
| 20 | B | s | 4 | 0.3212214974 |
| 21 | B | p | 0.08 | -0.07815710493 |
| 22 | B | p | 0.14 | 0.151709909 |
| 23 | C | l | 0.08 | -0.002618096088 |
| 24 | C | l | 0.1 | 0.02912160367 |
| 25 | C | l | 0.12 | -0.05193077358 |
| 26 | C | l | 0.2 | 0.08199428243 |
| 27 | C | l | 0.3 | -0.05899034117 |
| 28 | C | l | 0.4 | -0.1717704784 |
| 29 | C | l | 0.7 | 0.2890134877 |
| 30 | C | l | 1.6 | -0.6690791094 |
| 31 | C | s | 0.1 | 0.02641795942 |
| 32 | C | s | 0.3 | -0.08011918062 |
| 33 | C | s | 4 | 0.5135931759 |
| 34 | C | p | 0.08 | -0.04102337581 |
| 35 | C | p | 0.14 | 0.1543425571 |
| 36 | C | p | 0.16 | 0.005426003158 |
| 37 | N | l | 0.08 | 0.003916089049 |
| 38 | N | l | 0.12 | -0.01209604524 |
| 39 | N | l | 0.2 | 0.007052628297 |
| 40 | N | l | 0.22 | 0.008680367925 |
| 41 | N | l | 0.7 | -0.3205701579 |
| 42 | N | l | 3 | 0.2883822236 |
| 43 | N | s | 0.1 | 0.07318190533 |
| 44 | N | s | 0.5 | -0.04142680556 |
| 45 | N | p | 0.08 | -0.08742704787 |
| 46 | N | p | 0.14 | 0.2438428886 |
| 47 | N | p | 0.5 | -0.09068615632 |
| 48 | O | l | 0.08 | -0.02521325316 |
| 49 | O | l | 0.1 | 0.1154773506 |
| 50 | O | l | 0.12 | -0.131349275 |
| 51 | O | l | 0.2 | 0.07039514689 |
| 52 | O | l | 0.4 | -0.06218435634 |
| 53 | O | l | 1 | -0.4934296935 |
| 54 | O | s | 0.08 | 0.05183947158 |
| 55 | O | s | 0.4 | -0.0770530214 |
| 56 | O | s | 4 | 0.007657587066 |
| 57 | O | p | 0.08 | -0.0441084329 |
| 58 | O | p | 0.14 | 0.0104554608 |
| 59 | O | p | 0.16 | 0.1378003772 |
| 60 | O | p | 0.5 | -0.09853965851 |
| 61 | O | p | 4 | 0.5474151274 |
| 62 | F | l | 0.08 | -0.007651018086 |
| 63 | F | l | 0.1 | 0.02222382003 |
| 64 | F | l | 0.16 | -0.04503808767 |
| 65 | F | l | 0.28 | 0.05445365585 |
| 66 | F | l | 0.3 | 0.05587047966 |
| 67 | F | l | 0.7 | -0.2692779501 |
| 68 | F | l | 0.8 | -0.2184186297 |
| 69 | F | s | 0.08 | -0.03049268533 |
| 70 | F | s | 4 | 0.0825775194 |
| 71 | F | p | 0.18 | 0.05537506811 |
| 72 | Si | l | 0.08 | 0.005411854811 |
| 73 | Si | l | 0.18 | -5.225981042e-05 |
| 74 | Si | l | 0.2 | -0.03908705111 |
| 75 | Si | s | 0.08 | -0.1539039298 |
| 76 | Si | p | 0.18 | 0.06035911192 |
| 77 | Si | p | 3 | 0.4430728735 |
| 78 | P | l | 0.08 | 0.002079212916 |
| 79 | P | l | 0.14 | -0.002222868568 |
| 80 | P | l | 0.26 | -0.06195693945 |
| 81 | P | l | 0.28 | -0.03169678226 |
| 82 | P | s | 0.08 | 0.1021454874 |
| 83 | P | p | 0.08 | -0.05429454458 |
| 84 | P | p | 0.2 | 0.1155644567 |
| 85 | S | l | 0.08 | 0.00408545218 |
| 86 | S | l | 0.12 | -0.0019571135 |
| 87 | S | l | 0.22 | 0.004095989245 |
| 88 | S | l | 0.4 | -0.2787783137 |
| 89 | S | l | 0.5 | -0.04968880762 |
| 90 | S | s | 0.08 | 0.2903200571 |
| 91 | S | s | 4 | -0.2989405707 |
| 92 | S | p | 0.08 | -0.04775630073 |
| 93 | S | p | 0.1 | -0.2822775036 |
| 94 | S | p | 0.26 | 0.661892812 |
| 95 | Cl | l | 0.08 | 0.003474205329 |
| 96 | Cl | l | 0.1 | -0.009928237399 |
| 97 | Cl | l | 0.16 | 0.03173939248 |
| 98 | Cl | l | 0.3 | -0.2030771775 |
| 99 | Cl | s | 0.12 | 0.04205976024 |
| 100 | Cl | p | 0.08 | -0.04737429474 |
| 101 | Cl | p | 0.12 | 0.107433834 |

%% acp write bleh 002_acp_write.acp
* ACP WRITE: writing ACP bleh to file 002_acp_write.acp

